### PR TITLE
Origin/editorialreview/modeler

### PR DESCRIPTION
**Pull request by christinaausley**
An editorial review of all docs within the "modeler" folder for grammar, punctuation, organization, and terminology.

### CODE REVIEW

**Review by volkergersabeck — DISMISSED** [review]
I just reviewed the Camunda Modeler parts of this PR and those changes are fine with me. Anything regarding Cloud Modeler is outside my responsibility.

What I saw on top of these text changes: the screenshots are very outdated (and will be even more with the next Modeler release since we will ship a new properties panel that you can already see in the Modeler nightly builds: https://downloads.camunda.cloud/release/camunda-modeler/nightly/)

Ideally, we could introduce our screenshot automation (https://github.com/camunda/camunda-docs-modeler-screenshots) to this docs repo as well.

**Comment by christinaausley** [comment]
@volkergersabeck Excellent. Thanks for your quick review.

I've added @ultraschuppi and @urbanisierung for review of Cloud Modeler.

Additionally, this screenshot automation technique is super interesting. I've followed up with Amara and may ping you in Slack if we need additional details on implementation.

File: docs/product-manuals/modeler/cloud-modeler/launch-cloud-modeler.md [hunk 1]
```diff
@@ -3,6 +3,12 @@ id: launch-cloud-modeler
 title: Launch Cloud Modeler
 ---
 
-Since the Cloud Modeler is embedded in the Camunda Cloud Console, it does not need to be started separately. Go to the tab **Diagrams** and click the button **Create New Diagram** or click on an **existing diagram** to jump to the Modeler.
+Since Cloud Modeler is embedded in the Camunda Cloud Console, it does not need to be started separately.
+
+To launch Cloud Modeler, follow th steps below:
```
Comment (urbanisierung) [inline docs/product-manuals/modeler/cloud-modeler/launch-cloud-modeler.md:8]: typo: follow th`e` steps

**Review by urbanisierung — COMMENTED** [review]
@christinaausley great! just found a small typo. besides that lgtm!

**Comment by christinaausley** [comment]
@urbanisierung just adjusted that typo. Let me know when you've reviewed and we'll go ahead and merge.

**Review by urbanisierung — APPROVED** [review]
/lgtm

thanks @christinaausley!

**Comment by christinaausley** [comment]
Merging as Volker and Adam have both left their remarks. Please let me know if you'd like me to make any changes beyond as I'm happy to open another branch.

**Pull request merged**